packages/components/src/AtlantisContext/AtlantisContext.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { createContext, useContext } from "react";
+
+export interface AtlantisContextProps {
+  /**
+   * The date format Atlantis components would use
+   */
+  readonly dateFormat: string;
+
+  /**
+   * The time format Atlantis components would use
+   */
+  readonly timeFormat: string;
+
+  /**
+   * Time zone used in converting the date and time formats
+   */
+  readonly timeZone: string;
+
+  /**
+   * Grabs the decimal separator and group separator based on locale
+   */
+  readonly floatSeparators: Record<"decimal" | "group", string>;
+
+  /**
+   * The currency symbol Atlantis components will use
+   */
+  readonly currencySymbol: string;
+
+  /**
+   * The `headerHeight` property represents the height of the app header in Atlantis.
+   * It plays a crucial role in determining the positioning of various elements within the app.
+   * By accurately defining this value, Atlantis can effectively calculate the layout and alignment of its components.
+   */
+  readonly headerHeight: number;
```

Comment:
Header height should be removed because it is only used for stuff related to React Navigation in react native

Reply:
Done! Missed this one.